### ISSUE TITLE
`catalyst` is not in supported platform list of `upload_symbols_to_bugsnag`

### ISSUE THREAD

**Issue by son-nguyen-goodnotes** [bugsnag/bugsnag-dsym-upload#83]
### Describe the bug
When running `upload_symbols_to_bugsnag` for catalyst build, I got this error:
```
Action 'upload_symbols_to_bugsnag' isn't known to support operating system 'catalyst'.
```
Upon checking [this file](https://github.com/bugsnag/bugsnag-dsym-upload/blob/master/tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb), I see that it only support `[:ios, :mac, :tvos]`.

Please add catalyst to this as well.

Thanks

### Steps to reproduce
Run `upload_symbols_to_bugsnag` for catalyst

**Comment by clr182** [comment]
Hi @son-nguyen-goodnotes 

Thank you for reaching out and raising this issue. 

I've added this to our backlog for further investigation. We will be sure to fix this when priorities allow.

**Comment by mclack** [comment]
Hi @son-nguyen-goodnotes 

This has now been fixed in the v2.3.1 release of `bugsnag-dsym-upload`: https://github.com/bugsnag/bugsnag-dsym-upload/releases/tag/v2.3.1

Thanks again for raising this with us. Please let us know if you continue to have any issues after upgrading to the latest version.
